### PR TITLE
doc: update changelog for 2.10.0 release

### PR DESCRIPTION
**Pull request by Kaian**
Changelog for release 2.10.0

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+        - Added support for T.38 passthrough for Residential Devices
+
+    * Retail
+        - Added support for T.38 passthrough for Retail Accounts
+
+    * Invoices
+        - Fixed automatic invoice number assignment through invoice scheduler
+
+    * Other
+        - Tests: Increase PHPStan tests level to 2.5
```
Comment (mmadariaga) [inline ChangeLog]: This is not related to any feature, I would remove it

Comment (Kaian) [reply ChangeLog]: Removed!
